css/mobile.css

```diff
@@ -0,0 +1,402 @@
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 1.5rem;
+  right: 3rem;
+  font-size: 4rem;
+}
+
+.desktop-navbar {
+  display: none;
+}
+
+/* First Page */
+.container {
+  width: 100%;
+  margin: 0;
+}
+
+.main-content-headers h1 {
+  width: 100%;
+}
+
+.main-content-headers h2 {
+  width: 100%;
+  margin: 3% 0;
+}
+
+.container h1 {
+  font-family: "cocogoose", sans-serif;
+  font-size: 1.85rem;
+  letter-spacing: 1.5px;
+}
+
+.year {
+  font-family: "Lato", sans-serif;
+  margin-bottom: 15%;
+  width: 100%;
+}
+
+.header-content {
+  border: 1px solid #d3d3d3;
+  background-color: greenyellow;
+  padding: 3%;
+  line-height: 1.5rem;
+  backdrop-filter: saturate(130%);
+  opacity: 0.9;
+  box-shadow: 0 0 25px 0 inset #d3d3d3;
+  -webkit-box-shadow: 0 0 25px 0 inset #d3d3d3;
+  width: 100%;
+  font-size: 1.25rem;
+}
+
+.main-content .text-center h1 {
+  margin-top: 5%;
+}
+
+.main-content-headers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.background-header-image {
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+  bottom: 0;
+  left: 0;
+  background-size: cover;
+}
+
+.background-header-image-rotate {
+  transform: rotate(180deg);
+  top: 0;
+  right: 0;
+  position: absolute;
+  width: 100%;
+  z-index: -2;
+  background-size: cover;
+}
+
+.infos {
+  margin: 10% 0;
+}
+
+/* Main Program */
+.main-program {
+  background: url("https://jicamargo.github.io/mmw/images/background-x.png");
+  width: 100%;
+  height: 100%;
+  padding: 3%;
+  border-radius: 5px;
+  margin: 1%;
+}
+
+.main-program h2 {
+  text-decoration-color: white;
+}
+
+.program-item {
+  padding: 3%;
+  margin: 2% 0;
+  opacity: 0.7;
+  border-radius: 5px;
+  background-color: rgba(211, 211, 211, 0.1882352941);
+}
+
+.item-icon {
+  width: 12%;
+  color: #fff;
+}
+
+.item-header {
+  width: 28%;
+}
+
+.item-content {
+  width: 50%;
+  color: #fff;
+}
+
+.main-btn {
+  margin: 5% 0;
+  cursor: pointer;
+}
+
+.see-program {
+  display: none;
+}
+
+/* (This Years) Conference Speakers */
+.featured-speakers-header {
+  margin: 1%;
+}
+
+.speaker-heading {
+  margin-bottom: 2%;
+}
+
+.speaker-image {
+  width: 150px;
+  margin: 3%;
+}
+
+.speakers-list {
+  opacity: 0.8;
+}
+
+.speaker-infos {
+  width: 67%;
+  margin: 3%;
+  margin-left: 0;
+}
+
+.speakers-btn {
+  width: 80%;
+  border: #272a31 solid 1px;
+  background-color: #fff;
+  color: #272a31;
+  margin: 5% 0;
+}
+
+.down-arrow {
+  color: #ec5242;
+  margin: 0 5px;
+}
+
+.speakers-display {
+  display: none;
+}
+
+/* Partners */
+.partners {
+  width: 100%;
+  opacity: 0.9;
+  margin: 0;
+  background: #272a31;
+}
+
+.partners-heading {
+  margin-top: 10%;
+}
+
+.partners-list ul {
+  display: flex;
+  flex-direction: row;
+}
+
+.list-item-container {
+  margin: 10% 5%;
+}
+
+.list-item-container:hover {
+  opacity: 0.7;
+}
+
+.list-item-container::marker {
+  color: #272a31;
+}
+
+.list-item {
+  width: 80px;
+  height: 80px;
+}
```

Comment:
Kindly note rhat for the mobile view of your design, you should have a partners section in the about page. kindly implement this for your design

Your design :
![image](https://github.com/katarighe/microverse-capstone-project-1/assets/20537712/1c183cbc-6351-4eaa-9b14-35b9204ae2eb)

Design given 👍 
![image](https://github.com/katarighe/microverse-capstone-project-1/assets/20537712/04e3ab50-accc-4139-8c3b-4500844df979)